nextflow.config

```
@@ -73,8 +69,8 @@ params {
 
     // Schema validation default options
     validationFailUnrecognisedParams = false
-    validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base,monochromeLogs'
+    validationLenientMode            = true
```

Why true?

---

That is a leftover from fixing the other bug with the string/integer

---

Perhaps we revert it back since it's no longer needed.
```suggestion
    validationLenientMode            = false
```